requirements/release.txt

```diff
@@ -0,0 +1,11 @@
+# docs
```

Comment:
@radujica I separated again, but we can discuss

Reply:
looks good to me